app/src/main/java/org/dhis2/usescases/settings/SyncWorkInfoHelper.kt

```diff
@@ -0,0 +1,23 @@
+package org.dhis2.usescases.settings
+
+import androidx.work.WorkInfo
+
+object SyncWorkInfoHelper {
+    fun onWorkStatusesUpdate(
```

Comment:
I think that this is adding complexity when testing due to is not injected, I would use `SyncManagerPresenter` and also I would add tests